console/src/main/scripts/plugins/metrics/ts/metricsAppServerList.ts

```diff
@@ -21,6 +21,35 @@
 
 module HawkularMetrics {
 
+  export class ServerStatus {
```

Comment:
Maybe this should be in its own file as its generally useful in many places 'serverStatus.ts'